multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/services/ProcessLogsPersistenceService.java

```diff
@@ -70,6 +100,19 @@ private List<FileEntry> listFiles(final String space, final String operationId,
         }
     }
 
+    private List<OperationLogEntry> listOperationLogs(final String space, final String operationId, String logId)
+        throws FileStorageException {
+        try {
+            return getSqlQueryExecutor().execute(sqlOperationLogQueryProvider.getListFilesQueryBySpaceOperationIdAndLogId(space,
+                                                                                                                          operationId,
+                                                                                                                          logId));
+        } catch (SQLException e) {
+            throw new FileStorageException(MessageFormat.format(Messages.ERROR_GETTING_LOGS_WITH_SPACE_OPERATION_ID_AND_NAME, space,
+                                                                operationId, logId),
+                                           e);
+        }
+    }
+
     public void persistLog(String space, String operationId, File localLog, String remoteLogName) {
```

Comment:
I think this method is no longer used. Could you delete it as well as the constants that it uses